docs/dynamic-programming/unbounded-knapsack.mdx

```diff
@@ -0,0 +1,207 @@
+The values are  `50, 100`.
+We start from the nth item whose weight is 10 which is less than W. So we can either choose it or not depending on what will maximise the value of the knapsack.
+We get greater value when including the nth item than when we get by excluding it.
+The maximum value comes out to be `5*100=500`.
+```
+#### Example 2
+
+```
+Let capacity W be `5` and number of items be `2`.
+The weights are `3,5`.
+The values of items are `10,15`.
+We start from the nth item whose weight is equal to the knapsack W, so we may either include it or exclude it. If we include it the value of the knapsack will be `15`, but if we exclude it the value of the knapsack will be `10`. 
+So the maximum value of the knapsack will be `15`.
+```
+
+## Algorithm
+
+- Create a table (let it be called dp here). In this table `dp[i][j]` will represent the value of the knapsack if there are `i` items and the weight of the knapsack is `j`.
+- Identify the base conditions: if the weight of the knapsack is zero, but the number of items is not zero, then the value of knapsack will be zero and if the weight of the knapsack is not zero but the number of items is zero then value will be zero.
+- Through looping assign the values to each position in the table.
+- If the weight of the item is less than the weight of the knapsack then compare the value of the knapsack when the item is included with the value when the item is excluded.
+  If the item is included, then we again choose from the n items.
+- If the weight of item is greater than the weight of the knapsack then we have to exclude the item and choose from other n-1 items.
+- Return the value of the last position in the table as it is the maximum value of the knapsack.
+
+## Code
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs
+  defaultValue="cpp"
+  values={[
+    { label: 'C++', value: 'cpp', },
+    { label: 'Python', value: 'py', }
+  ]
+}>
+<TabItem value="cpp">
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int unboundedKnap(int w[], int v[], int W, int n)
+{
+    //table
+    int t[n + 1][W + 1];
+
+    /*Base conditions
+    If the number of items in the knapsack is zero, then the value of the items taht can be packed is zero
+    If the weight of the knapsack is zero, then the value of the items taht can be packed into the knapsack will be zero too*/
+    for (int i = 0; i < n + 1; i++)
+    {
+        for (int j = 0; j < W + 1; j++)
+        {
+            if (i == 0 || j == 0)
+                t[i][j] = 0;
+        }
+    }
+
+    /*-->If the weight of the ith item is less than the weight of the knapsack, then we can either include the item or exclude it depending upon which will maximise the value
+      -->If included then we can include it again
+      -->If not included once then it is not included again
+      -->If the weight is greater than the weight of the knapsack then we cannot include it into the knapsack at any time
```

Comment:
Don't use `-->`.  Keep them as ordered list

Reply:
Changed them to ordered list